content/docs/what-is-kudo.md

```diff
@@ -133,3 +133,38 @@ Normally, _Operator_ and _OperatorVersion_ are handled by the manager and rarely
 ```bash
 $ kubectl kudo install kafka --instance dev-kafka
 ```
+
+## Executing Plans
```

Comment:
Note that there already is an "Operator Plans" section above. It says that more on plans can be read in `/docs/developing-operators/plans.html` so adding more info about plans here is a bit of a hidden surprise. :-P

I think what you added actually belongs in that file not here, though it may make sense to add a few words to the above section just to mention that manual plan triggering is also possible.

Reply:
Agree. Done.